scripts/test/Muon/grouping_tab/grouping_tab_presenter_test.py

```diff
@@ -216,16 +219,6 @@ def test_update_all_calculates_groups_and_pairs(self):
                                                                                 self.presenter.error_callback)
         self.presenter.update_thread.start.assert_called_once_with()
 
-    def test_removing_group_removes_linked_pairs(self):
```

Comment:
I have deleted this in the interest of speed. In #30558 we will add tests and this one will be replaced by checking that a warning is produced.